tests/Integration/BuildTest.php

```diff
@@ -85,6 +85,7 @@ public function testSiteMap(): void
 
         self::assertEqualsCanonicalizing([
             'http://localhost/',
+            'http://localhost/authors/john.doe',
             'http://localhost/authors/ogi',
```

Comment:
Testons :`http://localhost/foo.html` qui devrais bien donner `foo.html` ?

Reply:
This way? https://github.com/Tom32i/content/compare/d611f8fa65c0998554d846db534ea1a9e39ac027..6b42d0c245891c698fa6d0222b56115a721ca6f1